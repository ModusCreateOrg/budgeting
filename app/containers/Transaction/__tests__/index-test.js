import React from 'react';
import renderer from 'react-test-renderer';
import { TransactionContainer } from '../index';

const mockMatch = {
  params: {id: null}
}
const mockHistory = {
  goBack(){}
}

it('renders correctly', () => {
  const tree = renderer.create(<TransactionContainer match={mockMatch} history={mockHistory} />).toJSON();
  expect(tree).toMatchSnapshot();
});
