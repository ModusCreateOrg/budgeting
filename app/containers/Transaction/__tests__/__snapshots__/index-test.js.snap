// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="transaction"
>
  <h3>
    
  </h3>
  <div>
    <button
      onClick={[Function]}
    >
      Back
    </button>
  </div>
  <h4>
    Transaction id null not found
  </h4>
</div>
`;
